env/ci.go

```diff
@@ -27,8 +27,8 @@ func loadCIInfo() ci {
 	}
 }
 
-var ciNameVars = []string{"CI_NAME", "APPVEYOR", "BUILDKITE", "CIRCLECI", "GITLAB_CI", "JENKINS_URL", "SEMAPHORE", "TDDIUM", "TRAVIS", "WERCKER"}
+var ciNameVars = []string{"CI_NAME", "CI", "APPVEYOR", "BUILDKITE", "CIRCLECI", "GITLAB_CI", "JENKINS_URL", "SEMAPHORE", "TDDIUM", "TRAVIS", "WERCKER"}
```

Comment:
I think the `CI` environment variable is set to `true` in other CI environments, like [travis](https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables). Could we move this to the end of the list so that other, more specific environment variable names are attempted first?